meta-resin-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor

```diff
@@ -3,6 +3,19 @@
 # shellcheck disable=SC1091
 . /usr/sbin/resin-vars
 
+# Handle apps.json existence
+# Keep this before computing SUPERVISOR_CONTAINER_IMAGE_ID
+APPSJSON=/mnt/data/apps.json
+if [ ! -f "$APPSJSON" ]; then
```

Comment:
This `if` isn't necessary, right? since the other two conditions (`-d` and `! -e`) already imply this

Reply:
It actually has a reason there. In most of the cases the file will be in place and that if will behave as an early exit. If we remove it, the most common path with evaluate two conditions.

Reply:
Makes sense :+1: